src/pfs/pfsutil/pfsutil.go

```diff
@@ -39,6 +39,13 @@ func NewBlock(hash string) *pfs.Block {
 	}
 }
 
+func NewDiff(repoName string, commitID string, shard uint64) *pfs.Diff {
+	return &pfs.Diff{
+		Commit: NewCommit(repoName, commitID),
+		Shard:  shard,
+	}
+}
+
```

Comment:
Convenience function I finally decided to add.
Actually I'm going to go through now and use this in some places.